assets/pages/contributions-landing/newContributionsLanding.scss

```diff
@@ -0,0 +1,402 @@
+// -----  gu-sass ----- //
+
+@import '~stylesheets/gu-sass/gu-sass';
+
+body {
+  background: white;
+  margin: 0;
+  min-height: 100vh;
+  padding: 0;
```

Comment:
Is this mostly taken care of by `reset-src` (which you get from importing `gu-sass`)?